docker-compose.yml

```diff
@@ -22,7 +22,7 @@ x-env-mapping: &env
 services:
   worker: &worker
     <<: *env
-    image: addons/addons-server:latest-py3
+    image: addons/addons-server:latest
```

Comment:
This requires us to push the python3 image to `addons/addons-server:latest`.

Reply:
@EnTeQuAk once I merge, can you take care of that bit please ? I'm still operating with limited quota.

Reply:
absolutely! Just ping me in IRC and I'll rebuild and upload most recent images.